src/main/java/statisticker/Statistics.java

```diff
@@ -1,10 +1,39 @@
 package statisticker;
 
 import java.util.List;
+import java.util.Collections;
 
-public class Statistics 
-{
-    public static Stats getStatistics(List<___> numbers) {
-        //implement the computation of statistics here
+public class Statistics {
+
+    public static class Stats {
+        public float average;
+        public float min;
+        public float max;
+
+        public Stats(float average, float min, float max) {
+            this.average = average;
+            this.min = min;
+            this.max = max;
+        }
+    }
+
+    public static Statistics.Stats getStatistics(List<Float> numbers) {
+        if (numbers == null || numbers.isEmpty()) {
```

Comment:
Like NaN is used to express a "not valid" result, some of the input numbers can also be "not valid" NaN.
What do you think the stats should be in that case? What information do you need to decide that?

Reply:
NaN normally means the data is missing. If some of the data is missing, is it ok to go ahead and report the stats on the available data? Or, as you suggest, is it better to report NaN, to avoid conveying a wrong picture?
To answer this, we need to ask: "what does the data represent?" 
If the data gives the weight of fruits grown in a farm: Then a few NaNs can be ignored, we can report on the remaining data.
If the data represents share price readings across days, then showing the maximum (like a "52-week high") may not be appropriate. Even if one value is missing, we need to report that the stats are not available.
>Moral of the story: It's not possible to write good code without knowing the domain. The technique of "thinking in terms of tests" is a great way to know the domain.